### ISSUE TITLE
Unable to find the main primevideo.com navigation section

### ISSUE THREAD

**Issue by CGarces** [Sandmann79/xbmc#268]
Using 0.6.7

The error message is 

> Unable to find the main primevideo.com navigation section

Same as #165 

The logs show this:

```
18:52:06.927 T:3864359792  NOTICE: [Amazon VOD] Service: Start
18:52:07.090 T:3864359792  NOTICE: [Amazon VOD] Service: End
18:52:07.232 T:3864359792 WARNING: CPythonInvoker(5, /storage/.kodi/addons/plugin.video.amazon-test/service.py): the python script "/storage/.kodi/addons/plugin.video.amazon-test/service.py" has left several classes in memory that we cou
ldn't clean up. The classes include: N9XBMCAddon7xbmcgui6DialogE,N9XBMCAddon9xbmcaddon5AddonE
18:55:12.935 T:3704611696   ERROR: GetDirectory - Error getting
18:55:14.448 T:3652305776   ERROR: Previous line repeats 6 times.
18:55:14.449 T:3652305776  NOTICE: [Amazon VOD] Version: 0.6.7
18:55:14.449 T:3652305776  NOTICE: [Amazon VOD] Unicode filename support: False
18:55:14.450 T:3652305776  NOTICE: [Amazon VOD] Locale: en-gb / Language: en
18:55:16.030 T:3652305776   ERROR: [Amazon VOD] Unable to find the main primevideo.com navigation section

```

Full log at
https://pastebin.com/nN51FHxf

**Comment by CGarces** [comment]
Fixed after manual update to 0.6.8

Auto update not work due #267

**Comment by Varstahl** [comment]
Duiplicate of #265, already fixed.
